public/views/addStudent.html

```diff
@@ -0,0 +1 @@
+Add a student using our invisible form:
```

Comment:
we haven't moved everything around yet. this is just a template file where the add student form will be moved to